packages/node_modules/@ciscospark/widget-meet/src/container.js

```diff
@@ -64,7 +68,9 @@ export class MeetWidget extends Component {
 
   shouldComponentUpdate(nextProps) {
     const props = this.props;
-    return nextProps.media !== props.media || nextProps.toPersonAvatar !== props.toPersonAvatar;
+    return nextProps.media !== props.media
```

Comment:
I may have added the rules into our local eslint rules... It just seemed a bit silly to not allow multiple lines here.